### PR TITLE
MINOR: Replace deprectated 'TIMESTAMPTOSTRING' with 'FORMAT_TIMESTAMP'

### PR DESCRIPTION
**Pull request by bbejeck**
### Description

Updated insert statements to use 'UNIX_TIMESTAMP()' to generate dates

This PR proposes to solve the "too old" date issue by using `UNIX_TIMESTAMP()` to generate dates for inserting the initial data for the tutorial.  Additionally, the tutorial uses a deprecated method `TIMESTAMPTOSTRING`, so it's been replaced with the recommended`FORMAT_TIMESTAMP` function.

### CODE REVIEW

File: _includes/tutorials/session-windows/ksql/code/tutorial-steps/dev/populate-stream.sql [hunk 1]
```diff
@@ -1,9 +1,10 @@
...
-INSERT INTO clicks (ip, timestamp, url) VALUES ('51.56.119.117','2019-07-18T10:01:36Z','/convallis/nunc/proin.jsp');
-INSERT INTO clicks (ip, timestamp, url) VALUES ('53.170.33.192','2019-07-18T10:02:00Z','/vestibulum/vestibulum/ante/ipsum/primis/in.json');
-INSERT INTO clicks (ip, timestamp, url) VALUES ('51.56.119.117','2019-07-18T11:03:21Z','/vehicula/consequat/morbi/a/ipsum/integer/a.jpg');
-INSERT INTO clicks (ip, timestamp, url) VALUES ('51.56.119.117','2019-07-18T11:03:50Z','/pede/venenatis.jsp');
-INSERT INTO clicks (ip, timestamp, url) VALUES ('53.170.33.192','2019-07-18T11:40:00Z','/nec/euismod/scelerisque/quam.xml');
-INSERT INTO clicks (ip, timestamp, url) VALUES ('53.170.33.192','2019-07-18T11:40:09Z','/ligula/nec/sem/duis.jsp');
+INSERT INTO clicks (ip, timestamp, url) VALUES ('51.56.119.117',FORMAT_TIMESTAMP(FROM_UNIXTIME(UNIX_TIMESTAMP()),'yyyy-MM-dd''T''HH:mm:ssX'),'/etiam/justo/etiam/pretium/iaculis.xml');
+INSERT INTO clicks (ip, timestamp, url) VALUES ('51.56.119.117',FORMAT_TIMESTAMP(FROM_UNIXTIME(UNIX_TIMESTAMP() + (60 * 1000)),'yyyy-MM-dd''T''HH:mm:ssX'),'/nullam/orci/pede/venenatis.json');
```
Comment (bbejeck) [inline _includes/tutorials/session-windows/ksql/code/tutorial-steps/dev/populate-stream.sql:2]: The tutorial needs timestamps at different intervals to create sessions.  The idea is to add time in milliseconds to the current time - here the format is `Seconds * 1000`

File: _includes/tutorials/session-windows/ksql/code/tutorial-steps/dev/populate-stream.sql [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
+INSERT INTO clicks (ip, timestamp, url) VALUES ('53.170.33.192',FORMAT_TIMESTAMP(FROM_UNIXTIME(UNIX_TIMESTAMP() + (91 * 1000)),'yyyy-MM-dd''T''HH:mm:ssX'),'/mauris/morbi/non.jpg');
+INSERT INTO clicks (ip, timestamp, url) VALUES ('51.56.119.117',FORMAT_TIMESTAMP(FROM_UNIXTIME(UNIX_TIMESTAMP() + (96 * 1000)),'yyyy-MM-dd''T''HH:mm:ssX'),'/convallis/nunc/proin.jsp');
+INSERT INTO clicks (ip, timestamp, url) VALUES ('53.170.33.192',FORMAT_TIMESTAMP(FROM_UNIXTIME(UNIX_TIMESTAMP() + (2 * 60 * 1000)),'yyyy-MM-dd''T''HH:mm:ssX'),'/vestibulum/vestibulum/ante/ipsum/primis/in.json');
```
Comment (bbejeck) [inline _includes/tutorials/session-windows/ksql/code/tutorial-steps/dev/populate-stream.sql:5]: (Minutes * Sixty Seconds * 1000)

**Review by davetroiano — CHANGES_REQUESTED** [review]
Should we do anything about stale dates with the `ksql-test-runner`? i.e., this part [here](https://developer.confluent.io/tutorials/create-session-windows/ksql.html#test-it)

I ran into this [here](https://github.com/confluentinc/kafka-tutorials/pull/1340)

Maybe the compromise is to let the `ksql-test-runner` / "Test it" section have stale dates (maybe update them to be August 2022 rather than July 2019 as part of this, but leave them hard coded?), but also add infinite retention to topics [here](https://github.com/confluentinc/kafka-tutorials/pull/1340) to prevent race condition build failures due to topic cleanup due to default retention period not being large enough.

**Comment by davetroiano** [comment]
oops I missed this:
>NOTE There's one issue remaining - the test data relies on hardcoded dates in a JSON file, so the tests for this PR will fail until we develop approach for creating test data

How about:
1. update the "dev" queries and expected outputs to not select timestamp / window boundaries
2. update the "test" input / output timestamps to be hardcoded but recent
3. add infinite retention so that we're safe against race condition build failures

That would get us to a happy place of improved quality and reliable build, right?

**Comment by bbejeck** [comment]
@davetroiano  - PR updated 

>Should we do anything about stale dates with the ksql-test-runner? i.e., this part [here](https://developer.confluent.io/tutorials/create-session-windows/ksql.html#test-it)

After spending a good deal of time on this, I think we should leave the dates for the test runner alone because updating requires a lot of manual work for little-to-no gain.

The timestamps used in the tutorial are set to use the current time so it should always work as expected for users working through the tutorial

File: settings.gradle [hunk 3]
```diff
@@ -25,7 +25,7 @@ def preDir = {
     return SKIP_SUBTREE
   }
 }
-
+// Dummy comment to force build
```
Comment (davetroiano) [inline settings.gradle:28]: remove

**Comment by bbejeck** [comment]
Only one test failure and it's unrelated

Test for this PR passed https://confluentinc.semaphoreci.com/jobs/d8c26926-d73d-4f55-acfe-0feb12b2628f

**Review by davetroiano — APPROVED** [review]
LGTM! just remove the dummy comment before merging

**Comment by davetroiano** [comment]
>After spending a good deal of time on this, I think we should leave the dates for the test runner alone because updating requires a lot of manual work for little-to-no gain.

@bbejeck sounds good, I agree

File: settings.gradle (hunk 3 above)
Comment (bbejeck) [reply settings.gradle:28]: ah yeah good catch

**Comment by bbejeck** [comment]
latest change unrelated to files for the tutorial so merging now

**Comment by bbejeck** [comment]
Thanks for the review @davetroiano!

**Pull request merged**